package.json

```diff
@@ -23,7 +23,8 @@
     "tslint": "tslint src/**/*.ts tests/**/*.ts",
     "sonar-scanner": "npm run tslint && npm run test && sonar-scanner",
     "coverage": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.*Test.ts\" --coverage",
```

Comment:
need to update to only call unit tests.

Reply:
update should be good, feel free to resolve